obstacle_stop_planner_refine/include/obstacle_stop_planner/vehicle.hpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OBSTACLE_STOP_PLANNER__VEHICLE_HPP_
+#define OBSTACLE_STOP_PLANNER__VEHICLE_HPP_
+
+#include "rclcpp/rclcpp.hpp"
+#include "vehicle_info_util/vehicle_info.hpp"
+#include "geometry_msgs/msg/transform_stamped.hpp"
+#include "geometry_msgs/msg/pose.hpp"
+#include "obstacle_stop_planner/util.hpp"
+
+namespace motion_planning {
+
+class VehicleInfo : public vehicle_info_util::VehicleInfo
```

Comment:
I'm sorry but this inheritance smells bad... Let's consider another way!

Reply:
Thanks. I follow lane_departure_checker's way. bd2cba7

Reply:
Great, thank you!